Updated Virtual Try-On notebook to use Gemini 2.5 for segmentation mask.

NSTiwari opened the pull request:
Hi @Giom-V, 

Updated Virtual Try-On notebook to use Gemini 2.5 for segmentation mask. Could you please review?

Giom-V commented:
@NSTiwari Thanks for the quick update! It really works well!
Can you just add this link after the main title
`<a target=\"_blank\" href=\"https://colab.research.google.com/github/google-gemini/cookbook/blob/main/examples/Virtual_Try_On.ipynb\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" height=30/>`.
And also run the formatting script so I can do the merge?

Thanks again!

NSTiwari commented:
> @NSTiwari Thanks for the quick update! It really works well! Can you just add this link after the main title `<a target=\"_blank\" href=\"https://colab.research.google.com/github/google-gemini/cookbook/blob/main/examples/Virtual_Try_On.ipynb\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" height=30/>`. And also run the formatting script so I can do the merge?
> 
> Thanks again!

@Giom-V done! You may please review and merge.

Few points:
1. I've moved the old implementation (using SAM 2.1) to my [personal repository](https://github.com/NSTiwari/Virtual-Try-On-Imagen3/blob/main/Virtual_Try_On_Imagen3.ipynb) and added its link to the current notebook.

2. The link to Virtual Try-On notebook in [Spatial_understanding.ipynb](https://colab.research.google.com/github/google-gemini/cookbook/blob/main/quickstarts/Spatial_understanding.ipynb#scrollTo=WQJTJ8wdGOKx) seems to be broken. Could you please fix it?
![image](https://github.com/user-attachments/assets/32ca225e-6e4b-4767-913a-1cc537392ce1)

Giom-V commented:
> 1. I've moved the old implementation (using SAM 2.1) to my [personal repository](https://github.com/NSTiwari/Virtual-Try-On-Imagen3/blob/main/Virtual_Try_On_Imagen3.ipynb) and added its link to the current notebook.

Good idea!

> 2. The link to Virtual Try-On notebook in [Spatial_understanding.ipynb](https://colab.research.google.com/github/google-gemini/cookbook/blob/main/quickstarts/Spatial_understanding.ipynb#scrollTo=WQJTJ8wdGOKx) seems to be broken. Could you please fix it?

Yes, good catch, I'll fix it.

Giom-V commented:
The notebook is still not formatted, but I'll fix that while fixing the link. Thanks for updating the notebook @NSTiwari!

NSTiwari commented:
> The notebook is still not formatted, but I'll fix that while fixing the link. Thanks for updating the notebook @NSTiwari!

Thank you for merging, @Giom-V. About formatting the notebook, I did run the following commands:

`!python3 -m pip install -U --user git+https://github.com/tensorflow/docs`

`!python3 -m tensorflow_docs.tools.nbfmt Virtual_Try_On.ipynb`

Am I missing something?
